docs/en/reference/second-level-cache.rst

```diff
@@ -280,23 +280,17 @@ level cache region.
     .. code-block:: php
 
         <?php
-        /**
-         * @Entity
-         * @Cache(usage="READ_ONLY", region="my_entity_region")
-         */
+        #[Entity]
+        #[Cache(usage: 'READ_ONLY', region: 'my_entity_region')]
         class Country
         {
-            /**
-             * @Id
-             * @GeneratedValue
-             * @Column(type="integer")
-             */
-            protected $id;
-
-            /**
-             * @Column(unique=true)
-             */
-            protected $name;
+            #[Id]
+            #[GeneratedValue]
+            #[Column(type: 'integer')]
```

Comment:
```suggestion
            #[Column]
```

We omit the type for the string property below and there's no particular reason we need to be more explicit here.